tests/unit/conftest.py

```diff
@@ -179,7 +179,9 @@ def httpretty_():
 @pytest.fixture
 def application_instance(pyramid_request):
     return factories.ApplicationInstance(
-        consumer_key=pyramid_request.lti_user.oauth_consumer_key,
+        consumer_key=pyramid_request.lti_user.oauth_consumer_key
+        if pyramid_request.lti_user
+        else None,
```

Comment:
```suggestion
        consumer_key=pyramid_request.lti_user.oauth_consumer_key,
```

Reply:
Sorry I was juggle to many branches here.